.github/workflows/build.yml

```
@@ -50,6 +50,9 @@ jobs:
         id: west-build-SHIELD_NAME-left
         with:
           args: 'build "-s zmk/app -b BOARD_NAME -- -DSHIELD=SHIELD_NAME_left -DZMK_CONFIG=/github/workspace/config"'
+      - name: KEYBOARD_TITLE DTS File
+        if: ${{ always() }}
```

Thoughts on only doing this for failures, e.g.:

```suggestion
        if: ${{ failure() }}
```

---

I'd originally had it on failure, but the UI it gives in GitHub Actions didn't make it immediately clear it was skipped on purpose, so I wasn't sure if it would cause confusion for some people ("Why was this step skipped? Is that bad?!?").

Its also of potential use for people who make custom changes to their board in their config (i.e. say they enable an encoder etc), since they can see in the DTS file `status = "okay"`, complementing the Kconfig files.

I don't feel that strongly about it, so can change if you think that is on failure would work better.

---

Yeah, works for me.